src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.card__img,
```

Comment:
It's not a good practice to zero out margins for all elements of the card. It's better to set margins for each element separately, based on the specific design requirements or reset styles for tags in separate file called `reset.scss` where u can reset all styles that u need